crabs/tracker/track_video.py

```diff
@@ -89,7 +89,7 @@ def prep_outputs(self):
             self.tracking_output_dir,
         ) = prep_csv_writer(self.args.output_dir, self.video_file_root)
 
-        if self.args.save_video:
+        if self.args.gt_path or self.args.save_video:
```

Comment:
Yes, that why it's `or` there. Because we might pass the gt_path but not save_video but we need to make the videowriter in both situation

Reply:
Ah wait, I disagree with this. I think we should only make a video writer if `save_video` is passed.

Since we may pass ground truth, but not want a video right?

Reply:
Yes, we can do that. I have no strong opinion on that. I thought as for now, we have not much gt, maybe we always want the video, but yes you are right. Especially if just want mota. Will make a change!

Reply:
cool, thank you loads!